plugin.video.massengeschmack/.gitignore

```diff
@@ -0,0 +1,27 @@
+# OS-specific files
```

Comment:
That's exactly the reason why I used to do ZIP pulls. I need my make-release.sh script, but I'm pretty sure you don't want to have it in the repository. That's why the .gitattributes file is there which makes sure neither make-release.sh, nor .gitignore nor the .gitattributes file itself is in the release.